components/dashboard/DashboardLayout.js

```diff
@@ -3,13 +3,14 @@ import SideBar from './SideBar';
 import Navbar from './Navbar';
 import PropTypes from 'prop-types';
 
-const DashboardLayout = ({ children, title }) => {
+const DashboardLayout = ({ children, title, showAlertHere }) => {
```

Comment:
As discussed, showAlertHere doesn't work here, we can pass in the alert message and type to show the right alert.

Reply:
noted